server/src/http/errorHandler.ts

```diff
@@ -0,0 +1,22 @@
+import Debug from 'debug';
```

Comment:
Can we be consistent with filenames so it starts with capital letter -> `ErrorHandler.ts` which contains handle function?

Reply:
this is a module import

Reply:
ahh OK, understood